pkg/kn/commands/service_get.go

```diff
@@ -67,3 +77,38 @@ func NewServiceGetCommand(p *KnParams) *cobra.Command {
 	serviceGetFlags.AddFlags(serviceGetCommand)
 	return serviceGetCommand
 }
+
+func getServiceInfo(args []string, namespace string, client serving.ServingV1alpha1Interface, cmd *cobra.Command) (interface{}, error) {
+
+	switch len(args) {
+	case 0:
+		service, err := client.Services(namespace).List(v1.ListOptions{})
+		if err != nil {
+			return nil, err
+		}
+		if len(service.Items) == 0 {
+			fmt.Fprintf(cmd.OutOrStdout(), "No resources found.\n")
+			return nil, nil
+		}
+		service.GetObjectKind().SetGroupVersionKind(schema.GroupVersionKind{
+			Group:   "knative.dev",
```

Comment:
see comment for `revision get`

Reply:
https://github.com/knative/client/pull/150#discussion_r289272069